pkgs/servers/monitoring/zabbix/1.8.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, pkgconfig, postgresql, curl, openssl, zlib }:
+
+let
+
+  version = "1.8.22";
```

Comment:
I don't think we need both zabbix versions. It also seems the old versions would not be maintained well.